MAUI/Picker/dealing-with-header-and-footer.md

```diff
@@ -0,0 +1,314 @@
+---
+layout: post
+title: Header and footer with .NET MAUI Picker control | Syncfusion
+description: Learn here all about header and footer with Syncfusion .NET MAUI Picker (SfPicker) control.
+platform: maui
+control: SfPicker
+documentation: ug
+---
+
+# Dealing with Header and Footer in .NET MAUI Picker (SfPicker)
+
+This section explains about the header and footer customization of picker control.
+
+## Enable or disable header
+
+SfPicker allows enabling or disabling the header section by setting  `SfPicker.HeaderView.Height` property grater than 0. Default value of `SfPicker.HeaderView.Height` property is 0.
```

Comment:
SfPicker allows enabling or disabling the header section by setting the `SfPicker.HeaderView.Height` property greater than 0. The default value of the `SfPicker.HeaderView.Height` property is 0.

Reply:
Content modified